.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+    "editor.formatOnSave": true,
```

Comment:
🛑 Jeff made the fair point that onSave stuff should probably be up to us to specify, which is fair... so both this an the eslint one should probably get pulled.

Reply:
Yeah, remove this.